Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -412,33 +405,7 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
                                         loadSucceeded = true;
                                         layout.addView(mVungleBannerAd);
```

Comment:
For MREC ad size we have to use addView(view, param) to force layout MREC view.

Reply:
VungleBanner should handle MREC size as well as any other banner size